build-parent/pom.xml

```diff
@@ -66,14 +66,6 @@
 
         <!-- MicroProfile TCK versions used to be defined here but have moved to the concrete tck modules -->
 
-        <!-- Panache needs to run the hibernate-jpamodelgen annotation processor,
-             and unfortunately annotation processors are not covered by dependency management
-             in kotlin-maven-plugin; see https://github.com/quarkusio/quarkus/issues/37477#issuecomment-1923662964
-         -->
-        <hibernate-orm.version>6.5.2.Final</hibernate-orm.version>
-        <!-- Antlr 4 is used by the PanacheQL parser but also needs to match the requirements of Hibernate ORM 6.x-->
-        <antlr.version>4.13.0</antlr.version>
```

Comment:
So from what I understand @gsmet is opposed to importing BOMs that contain more than just dependency management for the project itself, due to bad experience with that in the past. I.e. Hibernate ORM's BOM should only manage Hibernate ORM's own artifacts.
So, while Hibernate ORM's BOM managing antlr (and bytebuddy) would indeed help us, I suspect that won't fly.

Reply:
I see... got it 😃 👍🏻

Reply:
If it's just ByteBuddy and Antlr and we are positive that it will always be Hibernate ORM enforcing their version, I think we can do it (and revert it in case this assumption is not true anymore).
But we have to check the effective POM to make sure it doesn't enforce anything else (by inheriting from a parent for instance).

Reply:
That's what I meant, the Hibernate ORM BOM has a bunch of other dependencies in it: JBoss logging, JPA, Jandex, Classmate, Agroal, c3p0, ... No way you'll ever want that in Quarkus.